src/main/java/RemoveContactDialog.java

```diff
@@ -0,0 +1,28 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class RemoveContactDialog extends BasePage{
```

Comment:
its better to name this method "DeleteContactDialog"

Reply:
I`ll fix.